test/test_validate_assets.py

```diff
@@ -13,26 +13,27 @@
 
 
 @pytest.mark.parametrize(
-    "test_subdir,check_images,check_names,check_names_skip_pattern,expected",
+    "test_subdir,check_images,check_names,check_names_skip_pattern,check_dockerfile,expected",

```

Comment:
As recommended elsewhere, I think you can skip adding the `check_dockerfile` parameter here.